lib/roman_numerals.rb

```diff
@@ -0,0 +1,16 @@
+class RomanNumerals
+  DECIMALS = [1, 4, 5, 9, 10, 50, 90, 100, 900, 1000].reverse
+  ROMANS = ["I", "IV", "V", "IX", "X", "L", "XC", "C", "CM", "M"].reverse
+
+  def translate(decimal)
+    roman = ""
+    for i in 0..DECIMALS.length - 1
```

Comment:
Using a for in ruby should be extremely rare. Try (0...DECIMALS.length) instead (three dots is exclusive of the end value).